Offseason2022/src/main/java/frc/robot/commands/IntakingStop.java

```diff
@@ -4,25 +4,27 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.FCConsts.FCMode;
+import frc.robot.Constants.INConsts.INMode;
+import frc.robot.Constants.TCConsts.TCMode;
+import frc.robot.subsystems.FloorConveyor;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.TowerConveyor;
 
 /**
  *
  */
 public class IntakingStop extends SequentialCommandGroup
 {
-  public IntakingStop( )
+  public IntakingStop(Intake intake, FloorConveyor fConv, TowerConveyor tConv)
   {
+    setName("IntakingStop");
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+        // Also add parallel commands using the
+        //
+        new IntakeRun(intake, INMode.INTAKE_STOP), new FloorConveyorRun(FCMode.FCONVEYOR_STOP, fConv),

```

Comment:
since the code formatter is fighting us here, and we'd like to see sequential lines, there is a way to disable to formatter for a few lines. Place the keyword "@formatter:off" on its own line before the first command, and then "@formatter:on" on its own line after the last command. Then make it one command per line (so someone can read down and see the sequence).